manifest.yml

```diff
@@ -36,6 +36,7 @@ applications:
       TZ: America/New_York
       EASEY_AUTH_API_APP_STATUS: ((appStatus))
       EASEY_MAINTENANCE_BYPASS_USERS: ((maintenanceBypassUsers))
+      EASEY_AUTH_API_ENABLE_AUDIT_LOG: true
```

Comment:
This isn't using the `enableAuditLog` variable. It's set to `true` in every environment, and the default is true anyway, so it doesn't make a difference, but as it is now there is no need for the variable in the respective manifest-vars files.